framework/packages/abstract-client/src/account.rs

```diff
@@ -224,6 +225,26 @@ impl<Chain: CwEnv> Account<Chain> {
         Ok(governance.owner_address())
     }
 
+    /// Executes a [`CosmosMsg`] on the proxy of the account.
+    pub fn execute(
+        &self,
+        execute_msgs: Vec<CosmosMsg>,
```

Comment:
Can this be `Vec<impl Into<CosmosMsg>>`?

Reply:
or even  `impl IntoIterator<Item = Into<CosmosMsg>`